### ISSUE TITLE
Train without loading pre-training weights, but load pre-training weights altogether

### ISSUE THREAD

**Issue by PiBigStar5712** [ultralytics/ultralytics#7391]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

**I want to train without loading pre-training weights.**
**My code version is**`8.0.62`

![image](https://github.com/ultralytics/ultralytics/assets/102813129/06a5c389-55ca-4ba4-aff3-603ea886a127)
![image](https://github.com/ultralytics/ultralytics/assets/102813129/5786bb9e-48ce-4aad-9e15-67397daf38c2)

**Logically, all I had to do was train with pretrained set to False, and it wouldn't load the pre-trained weight. But at this point I set the model parameter to empty, and an error is reported.**

![image](https://github.com/ultralytics/ultralytics/assets/102813129/62d2b9d8-4246-4ba0-be0d-9e7a65012271)
![image](https://github.com/ultralytics/ultralytics/assets/102813129/a7332ea0-4fcf-4700-8d17-00755f5cc164)

**How should I modify it to get the effect I want？**

**Comment by glenn-jocher** [comment]
@PiBigStar5712 hello! To train a YOLOv8 model without loading pre-trained weights, you should set the `--weights` argument to `''` (empty string) when running your training command. This will initiate training from scratch. If you encounter any errors after setting `--weights` to `''`, please ensure that the rest of your training command is correctly formatted and that there are no other conflicting arguments.

For more detailed guidance on training modes and options, you can refer to our documentation on the Train mode. If the issue persists, feel free to share the complete error message and the exact command you're using, and we'll be happy to help you troubleshoot further. Happy training! 😊

**Comment by PiBigStar5712** [comment]
> @PiBigStar5712 hello! To train a YOLOv8 model without loading pre-trained weights, you should set the `--weights` argument to `''` (empty string) when running your training command. This will initiate training from scratch. If you encounter any errors after setting `--weights` to `''`, please ensure that the rest of your training command is correctly formatted and that there are no other conflicting arguments.
> 
> For more detailed guidance on training modes and options, you can refer to our documentation on the Train mode. If the issue persists, feel free to share the complete error message and the exact command you're using, and we'll be happy to help you troubleshoot further. Happy training! 😊

**Thank you for your reply.When I assign the model in the yaml file to yolov8n-seg.yaml and keep the pre-trained parameter to False, I can use train.py normal training.**

**Comment by glenn-jocher** [comment]
@PiBigStar5712 Great to hear that you're able to train using `yolov8n-seg.yaml` with the pre-trained parameter set to False. If you have any more questions or need further assistance as you continue with your training, don't hesitate to reach out. Keep up the good work! 👍
